db/postgres/init.sql

```diff
@@ -1,5 +1,28 @@
+CREATE TABLE car_brands (
+    brand_id uuid PRIMARY KEY,
+    brand VARCHAR(255)
+);
+
+CREATE TABLE car_classes (
+    class_id uuid PRIMARY KEY,
```

Comment:
why not name all ids "id"? would be more uniform

Reply:
fixed